### DISCUSSION TITLE
enable button

### DISCUSSION THREAD

**Discussion by bestec2** [zauberzeug/nicegui#293] [Q&A] [answered]
Hi, could someone help with  a props to enable button
 `.props('disabled=true')` works with no problems, I tried  `.props('disabled=false')`, `.props('enabled=true')`  with no luck.

**Comment by falkoschindler** [comment, marked as answer]
You can do that with `.props('disabled')` and `.props(remove='disabled')`.

That's because `.props('disabled')` sets the property "disabled" to `True` (without specifying "=true"). And removing the property effectively sets it to `False`.

**Important update:** In #560 we noticed that although using the HTML attribute `disabled` works, the Quasar prop is called `disable` (without "d"). On some elements this makes a difference. So if in doubt, use `disable`.

**Reply by bestec2** [reply]
Thank you for the super fast reply. It solved the problem.

**Comment by falkoschindler** [comment]
Note: Since 1.2.9 "disableable" input elements have `enable` and `disable` methods as well as an `enabled` property.
